packages/jsii-reflect/test/__snapshots__/tree.test.ts.snap

```diff
@@ -0,0 +1,3178 @@
+ │   ├─┬ [90mclass[39m [36mPower[39m
+ │   │ ├── [4mbase[24m: CompositeOperation
+ │   │ └─┬ [4mmembers[24m
+ │   │   ├─┬ <initializer>(base,pow) [90minitializer[39m
+ │   │   │ └─┬ [4mparameters[24m
+ │   │   │   ├─┬ base
+ │   │   │   │ └── [4mtype[24m: @scope/jsii-calc-lib.NumericValue
+ │   │   │   └─┬ pow
+ │   │   │     └── [4mtype[24m: @scope/jsii-calc-lib.NumericValue
+ │   │   ├─┬ base [90mproperty[39m
+ │   │   │ ├── [3mimmutable[23m
+ │   │   │ └── [4mtype[24m: @scope/jsii-calc-lib.NumericValue
+ │   │   ├─┬ expression [90mproperty[39m
+ │   │   │ ├── [3mimmutable[23m
+ │   │   │ └── [4mtype[24m: @scope/jsii-calc-lib.NumericValue
+ │   │   └─┬ pow [90mproperty[39m
+ │   │     ├── [3mimmutable[23m
+ │   │     └── [4mtype[24m: @scope/jsii-calc-lib.NumericValue
+ │   ├─┬ [90mclass[39m [36mPropertyNamedProperty[39m
+ │   │ └─┬ [4mmembers[24m
+ │   │   ├── <initializer>() [90minitializer[39m
+ │   │   ├─┬ property [90mproperty[39m
+ │   │   │ ├── [3mimmutable[23m
+ │   │   │ └── [4mtype[24m: string
+ │   │   └─┬ yetAnoterOne [90mproperty[39m
+ │   │     ├── [3mimmutable[23m
+ │   │     └── [4mtype[24m: boolean
+ │   ├─┬ [90mclass[39m [36mPublicClass[39m
+ │   │ └─┬ [4mmembers[24m
+ │   │   ├── <initializer>() [90minitializer[39m
+ │   │   └─┬ hello() [90mmethod[39m
+ │   │     └── [4mreturns[24m: void
+ │   ├─┬ [90mclass[39m [36mPythonReservedWords[39m
+ │   │ └─┬ [4mmembers[24m
+ │   │   ├── <initializer>() [90minitializer[39m
+ │   │   ├─┬ and() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ as() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ assert() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ async() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ await() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ break() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ class() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ continue() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ def() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ del() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ elif() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ else() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ except() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ finally() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ for() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ from() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ global() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ if() [90mmethod[39m
+ │   │   │ └── [4mreturns[24m: void
+ │   │   ├─┬ import() [90mmethod[39m
```

Comment:
Ahhhh.... what's all this being sneaked in? 😉

Reply:
Oh it's improved test coverage... Some of the code tweaks (to be safer) caused the branch coverage to go down a little... And this basically improves coverage of previously only uncovered code.